tutorials/efas/introduction/dcs5_copper_constant_properties_electrothermal.i

```diff
@@ -1473,7 +1473,7 @@ stack_with_powder = ${fparse ram_cc_sinter_punch_height + powder_height}
     type = GapFluxModelConduction
     temperature = potential
     boundary = top_punch_right
-    gap_conductivity = 6.67e-2  # S/m ceramaterials, through thickness for graphite foil: https://www.ceramaterials.com/wp-content/uploads/2022/01/GRAPHITE_FOIL_TDS_CM_01_22.pdf
+    gap_conductivity = 6.67e-2   # S/m ceramaterials, through thickness for graphite foil: https://www.ceramaterials.com/wp-content/uploads/2022/01/GRAPHITE_FOIL_TDS_CM_01_22.pdf
```

Comment:
This spacing change is the only change showing in this git commit. Correct file to modify however, so a positive change! Please add in the changes to the time step size and the pyrometer postprocessor x coordinate to this input file.